### PR TITLE
Don't cut off descenders in inbox filter + snippets

### PR DESCRIPTION
**Pull request by buoyad**
Just wrasslin' with styling

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready

File: shared/chat/inbox/index.desktop.js [hunk 1]
```diff
@@ -170,8 +171,8 @@ const _containerStyle = {
   backgroundColor: globalColors.blueGrey,
   contain: 'strict',
   height: '100%',
-  maxWidth: 260,
-  minWidth: 260,
+  maxWidth: inboxWidth,
```
Comment (nathunsmitty) [inline shared/chat/inbox/index.desktop.js]: is there a reason why we're setting min and max width as opposed to just width?

File: shared/chat/inbox/row/small-team/index.js [hunk 2]
```diff
@@ -134,30 +149,28 @@ class SmallTeam extends React.PureComponent<Props, State> {
...
-  container: {flexShrink: 0, height: RowSizes.smallRowHeight},
+  container: {flexShrink: 0, height: InboxSizes.smallRowHeight},
   conversationRow: {
     ...globalStyles.flexBoxColumn,
     flexGrow: 1,
     height: '100%',
     justifyContent: 'center',
     paddingLeft: 8,
     paddingRight: 8,
+    width: InboxSizes.inboxWidth - 48 - 16,
```
Comment (nathunsmitty) [inline shared/chat/inbox/row/small-team/index.js]: Where's the - 48 - 16 coming from? A comment would be helpful here

File: shared/chat/inbox/index.desktop.js (hunk 1 above)
Comment (buoyad) [reply shared/chat/inbox/index.desktop.js]: I'm not sure, I didn't want to modify these styles, just to factor `260` out into a constant

File: shared/chat/inbox/row/small-team/index.js (hunk 2 above)
Comment (buoyad) [reply shared/chat/inbox/row/small-team/index.js]: Hmm, this doesn't seem to be doing anything actually.. I'll get rid of it

**Comment by buoyad** [comment]
ok @nathunsmitty, updated with your comments. PTAL

**Pull request merged**